docs/site/components/mdx-components.tsx

```diff
@@ -101,10 +101,6 @@ const PlatformTabs = (props: React.PropsWithChildren<unknown>) => {
     CodeTabsNameMap.has(c.props.mdxType.toLowerCase())
   );
 
-  if (!router.isReady) {
```

Comment:
Good catch! I am trying to remember the reason for including this catch but it looks like its not needed anymore.